src/clang-cpp-frontend/clang_cpp_convert.cpp

```diff
@@ -826,8 +826,15 @@ bool clang_cpp_convertert::get_constructor_call(
    */
 
   // Calling base constructor from derived constructor
+  bool need_new_obj = false;
+
   if (new_expr.base_ctor_derived())
     gen_typecast_base_ctor_call(callee_decl, call, new_expr);
+  else if (
+    new_expr.get_bool("memberInit") &&
```

Comment:
Let's stick to the non-capitalized names delimited by underscores. Also, I believe, this should be prefixed by `#` in order to indicate that it's a "comment".